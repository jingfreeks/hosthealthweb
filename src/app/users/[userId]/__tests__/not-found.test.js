import { render } from "@testing-library/react";
import UserNofound from "../not-found";

describe("User not found Component", () => {
  it("renders User not found Component to match snaptshots", async() => {
    const { container } = render(<UserNofound />);
    expect(container).toMatchSnapshot();
  });
});
