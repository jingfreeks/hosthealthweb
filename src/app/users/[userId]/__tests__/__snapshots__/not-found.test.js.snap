// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User not found Component renders User not found Component to match snaptshots 1`] = `
<div>
  <h1>
    The requested user does not exist
  </h1>
</div>
`;
